src/System.Text.Json/ref/System.Text.Json.csproj

```diff
@@ -5,17 +5,23 @@
   <ItemGroup>
     <Compile Include="System.Text.Json.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' != 'netstandard' AND '$(TargetsNetFx)' != 'true'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp' OR '$(TargetGroup)' == 'uap'">
```

Comment:
nit: TargetFramework

Reply:
Dont we use TargetGroup with versionless and targetFramework with specific versions ?

Reply:
oh right. ignore that one.